From aeb1bb4ff5ffb1797ac6ccd2a401d33f4cf27f22 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Sun, 29 Nov 2020 06:09:11 +0100
Subject: [PATCH 251/351] usb: typec: anx7688: We may not want try_src

Try_src makes it needlessly harder to connect two pinephones together.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 drivers/usb/typec/anx7688.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/typec/anx7688.c b/drivers/usb/typec/anx7688.c
index 0b3dd40e6c6d..044786e00378 100644
--- a/drivers/usb/typec/anx7688.c
+++ b/drivers/usb/typec/anx7688.c
@@ -482,7 +482,7 @@ static int anx7688_connect(struct anx7688 *anx7688)
 		goto err_vconoff;
 
         /* auto_pd, try.src, try.sink, goto safe 5V */
-        ret = anx7688_reg_write(anx7688, ANX7688_REG_FEATURE_CTRL, 0x1e);
+        ret = anx7688_reg_write(anx7688, ANX7688_REG_FEATURE_CTRL, 0x1e & ~BIT(2)); // disable try_src
 	if (ret)
 		goto err_vconoff;
 
-- 
2.34.0

